app/js/game/DepictIt.cards.js

```diff
@@ -2,6 +2,26 @@ export const DepictItCards = [
     "Screaming inside your heart",
     "Guineapig lab technicians",
     "Bum bongos",
+    "Piggy bank",
+    "Money box",
+    "Guitar hero",
+    "Bullseye",
+    "Park life",
+    "Quarantine",
+    "Tea and biscuits",
+    "Cookies and Cream",
+    "Henry VIII",
+    "Cookie monster",
+    "Cold shower",
+    "Lets go fly a kite",
+    "Worldwar II",
```

Comment:
Love them all except Worldwar II, can we lose that one and then I'm happy to merge!